docker/test

```diff
@@ -25,6 +25,9 @@ while getopts "d:i:j:g:t:-:" opt; do
 			;;
 		-)
 			case "${OPTARG}" in
+			    any)
```

Comment:
thanks, help use tab for indent in the bash.

Reply:
Alright, should be fixed in the latest commit 👍